README.md

```diff
@@ -25,32 +26,26 @@
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+-Github flow knowledge.
+-Basic CSS and HTML.
+-Installation of files.
 
 ### Setup
+-To clone the repository run `[https://github.com/NATASHA-ct/MY-PORTFOLIO.git]` in your terminal
 
 ### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
+-Set up Github Actions
+-Set up linter > Lighthouse , Webhint ,Stylelint ,
+-Added Eslint
```

Comment:
- [ ] **[OPTIONAL]** If you are looking to add bullet points to your README, simply add a space after `-`, like so

  ```
  - Set up Github Actions
  - Set up linter > Lighthouse , Webhint ,Stylelint ,
  - Added Eslint
  ```

Reply:
Thank you @TedLivist for this one, i didn't know how to do it, I thought it was impossible